_data/communication.yml

```
@@ -227,30 +230,30 @@ websites:
     img: slack.png
     tfa:
       - sms
-      - software
+      - totp
     doc: https://get.slack.help/hc/en-us/articles/204509068
 
   - name: SocketLabs
     url: https://socketlabs.com/
     img: socketlabs.png
     tfa:
       - sms
-      - software
+      - totp
     doc: https://help.socketlabs.com/docs/two-step-verification
 
   - name: SparkPost
     url: https://sparkpost.com/
     img: sparkpost.png
     tfa:
-      - software
+      - totp
     doc: https://www.sparkpost.com/docs/my-account-and-profile/enabling-two-factor-authentication/
 
   - name: Stackfield
     url: https://www.stackfield.com/
     img: stackfield.png
     tfa:
-      - software
-    doc: https://en.stackfield.help/article/92-activating-deactivating-the-two-factor-authentication
+      - totp
+    doc: https://www.stackfield.com/help/two-factor-authentication-2032
```

```suggestion
    doc: https://www.stackfield.com/help/-2032
```
Shorter URL possible.